test/manual/cardano-node/FlakyTests.md

```diff
@@ -0,0 +1,9 @@
+# Run Flaky Tests
+
+Some tests might have been disabled in CI with the `flakyBecauseOf` helper.
+
+Run them locally using 
+
+```bash
+RUN_FLAKY_TESTS=1 stack test cardano-wallet:integration
```

Comment:
:+1:

Reply:
:thinking: would this work also with a windows test bundle? I guess it should, right?
```
set RUN_FLAKY_TESTS=1
cardano-wallet-test-integration.exe
```

Maybe could be added to the instruction.

Reply:
It would be also awesome to be able to run only flaky ones, but that's good enough I suppose :)

Reply:
> Maybe could be added to the instruction.

Added 👍 

> It would be also awesome to be able to run only flaky ones

If we convert `flakyBecauseOf` to a wrapper around `it` rather than a wrapper around `pendingWith` we can customise the title of the test, making it easy to use `--match`. Wasn't sure how needed that was though. We can do later.